image/image_test.go

```diff
@@ -1103,6 +1076,221 @@ func (s *imageSuite) TestNoLocalParallelSnapInstances(c *C) {
 	c.Assert(err, ErrorMatches, `cannot use snap "foo_instance", parallel snap instances are unsupported`)
 }
 
+func (s *imageSuite) TestBootstrapWithKernelTrack(c *C) {
+	restore := image.MockTrusted(s.storeSigning.Trusted)
+	defer restore()
+
+	// replace model with a model that uses core18
+	rawmodel, err := s.brandSigning.Sign(asserts.ModelType, map[string]interface{}{
+		"series":       "16",
+		"authority-id": "my-brand",
+		"brand-id":     "my-brand",
+		"model":        "my-model",
+		"architecture": "amd64",
+		"gadget":       "pc",
+		"kernel":       "pc-kernel",
+		"kernel-track": "4.15",
```

Comment:
We need to talk about kernel tracks. We most likely can't have "4.15" as a track, since that would prevent smooth upgrades from 4.15 to, say, 4.16, but that is not really a relevant delta. We need to think about these channels as streams of compatibility that we need to keep, and then name the track after that. Let's talk somewhere live to just agree on something quickly.

Reply:
Makes sense, I changed the test to use "18" as the current kernel-track.